Xamarin.Forms.Core/Frame.cs

```diff
@@ -11,6 +11,8 @@ public class Frame : ContentView, IElementConfiguration<Frame>
 
 		public static readonly BindableProperty HasShadowProperty = BindableProperty.Create("HasShadow", typeof(bool), typeof(Frame), true);
 
+		public static readonly BindableProperty CornerRadiusProperty = BindableProperty.Create(nameof(CornerRadius), typeof(float), typeof(Frame), -1.0f);
```

Comment:
Per "Behavioral Changes", "-1" is the default value and it assures backward compatibility with existing apps. The previous implementation used different values on each platform.
I don't think "-1" can be interpreted as "don't use a corner radius". Being a corner radius, -1 seems like an invalid value anyway. It should make sense that "0" value is for no corner radius. 

To have backward compatibility, my options for default value were a nullable float and -1. I always preferred nullable, I used it for a long time, but for some reason, seeing recently other people's code, I started to think about using magic consts. And I don't like magic consts :) They need an extra constant field. On the other hand, nullable types are a little bit harder to use. Maybe I'm overthinking. 

Looking forward to hear your thoughts..